docs/source/installation/2-manual-install.rst

```diff
@@ -450,7 +450,13 @@ Install and Configure CommCare Cloud
 
     ::
 
-       $ echo "source ~/.commcare-cloud/load_config.sh" >> ~/.profile
+        $ nano ~/.profile
+
+    Paste the following before the line that sources the init-ansible script.
+
+    ::
+
+        source ~/.commcare-cloud/load_config.sh
```

Comment:
Any specific case where this approach better than the previous one?

Reply:
Based on the order of instructions, `load_config.sh` was being run _after_ `init-ansible`, but load_config plays a critical role in exporting the COMMCARE_CLOUD_ENVIRONMENTS variable to point to the right environments directory. I know this change complicates the setup a bit, but hopefully results in fewer issues down the road. Does that answer your question?

Reply:
Another option would be to have the `init-ansible` script `source ~/.commcare-cloud/load_config.sh` if it exists, and skip the instruction to add it to `.profile`.

Reply:
Yeah upon closer inspection, I may have incorrectly assumed this was the cause of an issue I was facing. It seems fair to say that the init-ansible script does not use the COMMCARE_CLOUD_ENVIRONMENTS variable for anything other than how it is used in `update_code.sh`. I think I know where I went wrong, but I'll try to run through the setup steps without this change just to make sure.

Reply:
Ok it seems fine. Reverted in 17240b4e210f6b9867fc35e9cd6f03f559100815.